cluster-autoscaler/cloudprovider/gce/gce_cloud_provider.go

```diff
@@ -200,6 +200,12 @@ func (gce *GceCloudProvider) Refresh() error {
 	return gce.gceManager.Refresh()
 }
 
+// GetProjectInfo returns the project id, location and cluster name.
+func (gce *GceCloudProvider) GetProjectInfo() (projectId, location, clusterName string) {
```

Comment:
How are you going to use this function as it is not part of CloudProvider interface?

Reply:
This should not be added to cloudprovider interface. If you need such method in your extension you will need to use type assertion.